### PR TITLE
GUI: take out some unwise shortcuts

### LINKED ISSUE #10161: [Feature Unrequest] Remove CTRL+S from the hotkeys
So as it turns out, to my absolute delight, pressing CTRL + S closes your game.

...

Why? *Why?*

I was in the middle of making a new mouse hook demonstration, meaning I wanted to get ahead of myself, skip the deref chain tracing, and just hardcode in some memory addresses for KAMI to hook onto. In this case however, that meant a grand total of 9 addresses, which took a great pain to gather. Didn't help that GTHD is very sensitive to pausing, so I always ran the chance of it hanging on me.

Then I made the mistake of accidentally pressing CTRL+S twice, by reflex, after switching away from Visual Studio to the game window. You may guess what happened.

Who thought this was a good idea? Why? Has that person not heard of Alt+F4 before? In what world is it a good idea to bind unconditional exit to the same button as most other applications' save feature?

My absolute best wishes to the OP of #2831, thank you for pissing me *way* off on this beautiful early morning.

### PR DESCRIPTION
**Pull request by dio-gh**
Prevents loss of work, hair, and life expectancy when multitasking with a text editor on the side. Happens rather often.

Fixes #10161
Shows a middle finger to #2831

### CODE REVIEW

**Comment by elad335** [comment]
Remove ctrl+R as well (restart emulation). Why? I think these "unrevertable actions" should not be accesible from secret keyboard shortcuts. Unlike ctrl+p (pause) for example which at worst case when hit, you learn about a new feature of the emulator and continue your gameplay.

**Comment by dio-gh** [comment]
I wholeheartedly approve.

**Comment by illusion0001** [comment]
Ctrl Q for quit and W for restart

File: rpcs3/rpcs3qt/gs_frame.cpp [hunk 1]
```diff
@@ -187,20 +187,6 @@ void gs_frame::keyPressEvent(QKeyEvent *keyEvent)
...
 		}
 		break;
-	case Qt::Key_S:
-		if (keyEvent->modifiers() == Qt::ControlModifier && !m_disable_kb_hotkeys && !Emu.IsStopped())
-		{
-			Emu.Stop();
-			return;
-		}
-		break;
-	case Qt::Key_R:
```
Comment (elad335) [inline rpcs3/rpcs3qt/gs_frame.cpp:197]: ctrl+R can mean now what ctrl+E currently does which is Resume emulation.

Comment (dio-gh) [reply rpcs3/rpcs3qt/gs_frame.cpp:197]: ill do it tomorrow, would turn into a mess if i kept relying on github's web editor

**Comment by dio-gh** [comment]
> Ctrl Q for quit and W for restart

Not happening.

File: rpcs3/rpcs3qt/gs_frame.cpp (hunk 1 above)
Comment (dio-gh) [reply rpcs3/rpcs3qt/gs_frame.cpp:197]: done

**Comment by dio-gh** [comment]
Ready for review I 'spose.

**Comment by Megamouse** [comment]
So you modify the Qt shortcuts but don't modify the main_window::keyPressEvent ?

**Comment by dio-gh** [comment]
As if I knew the codebase lol. Well, this is what reviews are for innit? Fixed.

**Comment by dio-gh** [comment]
Grepped through the folders, this should be all of it.

**Comment by Megamouse** [comment]
lgtm. can't test right now though

**Pull request merged**